indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -70,7 +70,7 @@
 
 {% macro render_data(registration, field, from_management=false) %}
     {% set data = registration.data_by_field -%}
-    {% set friendly_data = data[field.id].friendly_data if field.id in data else '' -%}
+    {% set friendly_data = data[field.id].get_friendly_data(for_humans=True) if field.id in data else '' -%}
```

Comment:
lowercase `true` (same for `false` and `none`) in templates (even though both work - lowercase is preferred)